### ISSUE TITLE
Update golang versions

### ISSUE THREAD

**Issue by jodh-intel** [kata-containers/runtime#1338]
The [versions database](https://github.com/kata-containers/runtime/blob/master/versions.yaml) currently specifies:

```sh
$ yq read versions.yaml languages.golang.version
1.10.4
$ yq read versions.yaml languages.golang.meta.newest-version
1.11.1
```

FWICS...

- The snap build is using golang 1.10.4:

  https://github.com/kata-containers/packaging/blob/master/snap/snapcraft.yaml#L16

- The packaging is using golang 1.11.1:

  https://github.com/kata-containers/packaging/blob/master/obs-packaging/gen_versions_txt.sh#L49

- The CI system is using golang 1.11.1:

  https://github.com/kata-containers/tests/blob/master/.ci/install_go.sh#L79

However, from https://golang.org/dl:

- The latest stable golang is now: 1.12.
- The oldest stable golang version is now: 1.11.5

# TL;DR

We should consider updating golang versions to:

```yaml
languages.golang.version = 1.11.5
languages.golang.meta.newest-version = 1.12
```

Related to discussions about `go mod` (#1331).

/cc @chavafg.
